### PR TITLE
Stage ONT data for delivery

### PR DESCRIPTION
**Pull request by ssjunnebo**
- Given a PID, look for project dirs in DATA, ANALYSIS and ONT_TAR
- Make a staging directory “PID/TIMESTAMP” in DELIVERY
- Symlink all the data to the staging dir

### CODE REVIEW

File: taca/delivery/delivery_classes.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+    def make_staging_path(self):
+        """Create a staging dir."""
+        timestamp = datetime.now().strftime("%Y%m%d%H%M")
+        self.project_staging_path = os.path.join(
+            self.staging_path, self.project_id, timestamp
+        )
+        try:
+            os.makedirs(self.project_staging_path)
+        except OSError as e:
```
Comment (aanil) [inline taca/delivery/delivery_classes.py:42]: I'm guessing this is to catch if the directory already exists?

File: taca/delivery/cli.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+
+# Upload data
+@delivery.command()
+@click.option(
+    "-d",
+    "--dds_id",
+    type=str,
+    required=True,
+    help="DDS PID generated by the stage command",
```
Comment (aanil) [inline taca/delivery/cli.py:43]: Will the DDS PID still be generated by the stage command?

**Review by aanil — APPROVED** [review]
👍

File: taca/delivery/cli.py (hunk 2 above)
Comment (ssjunnebo) [reply taca/delivery/cli.py:43]: No, you're right, I missed this. Thanks for spotting it 👍

**Pull request merged**